examples/event_handling/EventHandling.cpp

```diff
@@ -0,0 +1,224 @@
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Graphics.hpp>
+
+#include <vector>
+
+
+namespace
+{
+std::string vec2ToString(const sf::Vector2i& vec2)
+{
+    return '(' + std::to_string(vec2.x) + ", " + std::to_string(vec2.y) + ')';
+};
+} // namespace
```

Comment:
Not really related to this PR, but I wouldn't be against finally adding simple things like string conversion to SFML classes like Vector2D or Vector3D (i.e. where it makes sense and does more than default `std::to_string()`.

Reply:
https://github.com/SFML/SFML/issues/2128

I tried (and failed) to get support for that.